db/seeds.rb

```diff
@@ -0,0 +1,26 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+House.create!({name: "Stark", img_url: "http://i.imgur.com/FosCtcN.png"})
+
+House.create!({name: "Tyrell", img_url: "http://i.imgur.com/rc746MG.png"})
+
+House.create!({name: "Lannister", img_url: "http://i.imgur.com/JzP5h1D.png"})
+
+House.create!({name: "Bolton", img_url: "http://i.imgur.com/s75L4qp.png"})
+
+
+
+
+Character.create!({name: "Arya", alive: true, house_id: 1})
```

Comment:
Since you're not destroying everything at the start of your seed file, it could be dangerous to hard-code foreign keys in here. Would recommend saving the houses to variables and using those to establish relationships in your characters. So something like...

```rb
stark = House.create!({name: "Stark", img_url: "http://i.imgur.com/FosCtcN.png"})
Character.create!({name: "Arya", alive: true, house: stark})
```